snow/networking/router/chain_router_test.go

```diff
@@ -70,7 +78,15 @@ func TestShutdown(t *testing.T) {
 func TestShutdownTimesOut(t *testing.T) {
 	tm := timeout.Manager{}
 	// Ensure that the MultiPut request does not timeout
-	tm.Initialize("", prometheus.NewRegistry())
+	tm.Initialize(
```

Comment:
We should probably just pass an arg struct rather than passing args like this

Reply:
Added a config.